modules/introduction/partials/new-features-76.adoc

```diff
@@ -43,6 +43,10 @@ When restoring a backup, `cbbackupmgr` defaults to not overwriting existing user
 You can change this default behavior using the new `--overwrite-users` command-line argument. 
 See  xref:backup-restore:cbbackupmgr-config.adoc[cbbackupmgr config] and xref:backup-restore:cbbackupmgr-restore.adoc[cbbackupmgr restore] for more about user backup.
 
+* `cbbackupmgr` now encrypts the backup data before sending it to archive storage.
```

Comment:
The above statement is incorrect.  The cbbackupmgr encrypted backups is an optional feature, available to both cbbackupmgr CLI and the Backup Service.  Previous to 7.6, the feature was in Developer Preview.  Please provide a more detail description of the feature so that customers are aware -- this is a feature that many customers have been asking for.  Also, please provide a link to the documentation of the feature so that the customers know how to configure cbbackupmgr CLI  if they wish to use the encrypted backups feature. 

Encrypted Backups
The cbbackupmgr encrypted backups feature, where the backup client, the cbbackupmgr, encrypts the data before it is sent to the archive storage, is GA in 7.6.  Encrypted backups feature is available with both cbbackupmgr CLI and the Backup Service. See [cbbackupmgr encryption](https://docs.couchbase.com/server/current/backup-restore/cbbackupmgr-encryption.html).